i18n/translations/en.json

```diff
@@ -412,9 +412,11 @@
 	"languages.secondary.empty": "There are no secondary languages yet",
 
 	"license": "License",
+	"license.activate": "Activate it now",
 	"license.buy": "Buy a license",
 	"license.register": "Register",
```

Comment:
We still have a lot of "Register" wording in the Panel, especially in the registration dialog but also in the system view (license status). Do we want to bring them in line or is it intentional?

Reply:
I think we need to clean this up somehow, but it's maybe too much for this PR?

Reply:
#5922